src/views/mfa-verify/PassCodeForm.js

```diff
@@ -10,11 +10,17 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 /* eslint complexity: [2, 7] */
-define(['okta', 'q', 'views/shared/TextBox'], function (Okta, Q, TextBox) {
+define(['okta', 'q', 'views/shared/TextBox', 'util/Util'], function (Okta, Q, TextBox, Util) {
```

Comment:
I don't think you need `Util` anymore

Reply:
you are right since we removed CallAfterTimeout, util is no longer needed